src/Commands/Sql/SqlServerLister.cs

```diff
@@ -0,0 +1,43 @@
+using Azure.Identity;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Sql;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace AzureMcp.Commands.Sql;
+
+public sealed class SqlServerLister() : ISqlServerLister
```

Comment:
It seems to me we should have a SqlDatabaseService and this should be part of the service.